### ISSUE TITLE
Show on type

### ISSUE THREAD

**Issue by komxvl** [shentao/vue-multiselect#1244]
Hi! Amazing library, thanks! Could you help me please. If it is possible to create something[ like this](https://codesandbox.io/s/m59xvwlpkj?file=/example.js). What I mean: disabled showing dropdown on focus and showing list when type 1 or more symbols. Thanks for your time!

**Comment by Zyberg** [comment]
Yup, this is possible, but you will have to play around a little bit with css styles. I've tried a few things and it seems that deactivating the multiselect makes the search input box not responsive, hence it's better to have a class `.hide { display: none; }` and use the amazing js api with classList ;) (don't use .style.display on the element, since it's how the component shows / hides the dropdown, hence all your logic would be overwritten!)
Basically you hook up into @open and @search-change events, then call a function that determines whether to hide or show the input with `this.$refs.multiselect.$el.classList.add('.hide')` or `this.$refs.multiselect.$el.classList.remove('.hide')`. I am attaching fiddle with my sloppy implementation, but you can probably refine it a little bit more :)

https://jsfiddle.net/tocxz0uk/

**Comment by komxvl** [comment]
Great explanation and an amazing example! I just changed your code, added `this.searchQuery <= 2` .
Thank you, sir!
